woocommerce/woocommerce-gateway-stripe
# Wrap account summary API in WP_REST_Response

Fixes #2268

## Changes proposed in this Pull Request:

All of the API responses should be wrapped in `WP_Rest_Response`, but the response on the account summary API was missing it. This PR wraps the response array in `WP_Rest_Response`.

cc @allendav @joshheald 

## Testing instructions

Make a GET request to `/wp-json/wc/v3/wc_stripe/account/summary` (e.g. by running `curl --user username:password 'http://YOUR_SITE/wp-json/wc/v3/wc_stripe/account/summary'` with [Basic Auth](https://github.com/WP-API/Basic-Auth) installed). Verify that you get back a valid JSON response with account details.

---


**Post merge**
